docs-site/docs/development/development_processes/code_review.md

```diff
@@ -0,0 +1,131 @@
+
+This checklist encourages the authors, reviewers, and maintainers of pull requests (MRs) to confirm changes were analyzed for high-impact risks to quality, performance, reliability, security, observability, and maintainability.
+
+Using checklists improves quality in software engineering. This checklist is a straightforward tool to support and bolster the skills of contributors to the IRIDA Next codebase.
+
+#### Quality
+
+1. You have self-reviewed this PR per [code review guidelines](./code_review).
+1. For the code that this change impacts, you beloeve that the automated tests validated functionality that is highly important to users.
+1. If the existing automated tests do not cover the above functionality, you have added the necessary tests or added an issue to describe the automation testing gap and linked it to this PR.
+1. You have considered the technical aspects of this change's impact on IRIDA Next.
+1. You have considered the impact of this change on the frontend, backend, and database portions of the system where appropriate and applied the `~ux`, `~frontend`, `~backend`, and `~database` labels accordingly.
+
+#### Performance, reliability, and availability
+
+1. You are confident that this PR does not harm performance, or you have asked a reviewer to help assess the performance impact.
+1. You have added information for database reviewers in the PR description, or you have decided that it is unnecessary.
+1. You have considered the scalability risk based on future predicted growth.
+
+#### Documentation
+
+1. You have added/updated documentation or decided that documentation changes are unnecessary for this PR.
+
+#### Security
+
+1. You have confirmed that if thsi PR contains a change to processing or storing credentials or tokens, authorization and authentication methods, you have added the `~security` label.
+
+## Best practices
+
+### Everyone
+
+* Be kind.
+* Accept that many programming decisions are opinions. Discuss tradeoffs, which you prefer, and reach a resolution quickly.
+* Ask questions; don't make demands. ("What do you think about naming this `:sample_id`?")
+* Ask for clarification. ("I didn't understand. Can you clarify?")
+* Avoid selective ownership of code. ("mine", "not mine", "yours")
+* Avoid using terms that could be seen as referring to personal traits. ("dumb", "stupid"). Assume everyone is intelligent and well-meaning.
+* Be explicit. Remember people don't always understand your intentions online.
+* Be humble. ("I'm not sure - let's look it up.")
+* Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+* Be careful about the use of sarcasm. Everything we do is public; what seems like good-natured ribbing to you and a long-time colleague might come off as mean and unwelcoming to a person new to the project.
+* Consider one-on-one chats or video valls if there are too many "I didn't understand" or "Alternative solution:" comments. Post a follow-up comment summarizing one-on-one discussion.
```

Comment:
Typo in `calls`

Reply:
Addressed in https://github.com/phac-nml/irida-next/pull/105/commits/aa95bea4d612c311ca86792f706dd567be19251c